modules/cluster-logging-collector-pod-location.adoc

```diff
@@ -5,11 +5,11 @@
 [id="cluster-logging-collector-pod-location_{context}"]
 = Viewing logging collector pods
 
-You can use the `oc get pods  --all-namespaces -o wide` command to see the nodes where the Fluentd are deployed.
+You can view the Fluentd logging collector pods and the corresponding nodes that they are running on. The Fluentd logging collector pods run only in the `openshift-logging` project.
 
 .Procedure
 
-Run the following command in the `openshift-logging` project:
+* Run the following command in the `openshift-logging` project to view the Fluentd logging collector pods and its details:
```

Comment:
You could replace "its details" with "their details", since the sentence refers to multiple pods.

Reply:
Done.